src/plugins/gauge/components/GaugeComponent.vue

```diff
@@ -402,10 +402,10 @@ export default {
       const CHAR_THRESHOLD = 3;
       const START_PERC = 8.5;
       const REDUCE_PERC = 0.8;
-      const RANGE_CHARS_MAX = Math.max(
-        this.rangeLow.toString().length,
-        this.rangeHigh.toString().length
-      );
+      const RANGE_CHARS_MAX =
+        this.rangeLow && this.rangeHigh
+          ? Math.max(this.rangeLow.toString().length, this.rangeHigh.toString().length)
```

Comment:
Could this just be

`Math.max(Number(this.rangeLow), Number(this.rangeHigh))` ? Or am I misunderstanding

Reply:
I need the max of the number of characters, not the numbers themselves. For example, with 10 and -10, Math.max would give me "10" - but "-10" has more characters. Also, if rangeLow is or High are null, I have to check that first.